package.json

```diff
@@ -32,7 +32,7 @@
     "minimist": "^1.1.1"
   },
   "optionalDependencies": {
-    "appdmg": "^0.6.0"
+    "appdmg": "~0.6.4"
```

Comment:
Why ~ instead of ^?

Reply:
Oops. I got mixed up with my operators. Let me change that around.

Reply:
[56d9863](https://github.com/electron-userland/electron-installer-dmg/pull/132/commits/56d98633aab17d85f53fb74a0bc792f2b2777803)